### PR TITLE
Update changelog for v1.2.0

### PR DESCRIPTION
**Pull request by iandees**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+v1.2.0
+------
+* Roll back the use of psycopg2 connection pools. (See [tilequeue/#149](https://github.com/tilezen/tilequeue/pull/149) and [#62](https://github.com/tilezen/tileserver/pull/62))
+* Try increasing the maximum connection count on connection pools. (See [#61](https://github.com/tilezen/tileserver/pull/61))
```
Comment (nvkelso) [inline CHANGELOG.md]: `Try increasing` to `Increase`

Comment (nvkelso) [reply CHANGELOG.md]: Though did this change stay in after #62?

**Review by nvkelso — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (iandees) [reply CHANGELOG.md]: No, that's why I used "Try"

Comment (nvkelso) [reply CHANGELOG.md]: If we're not releasing this change by tagging 1.2.0, let's not mention it in the changelog.

Comment (iandees) [reply CHANGELOG.md]: Ok. I'm used to putting all changes between two tags in the changelog, regardless of if the resulting code is in there. Happy to not include it though.

**Comment by iandees** [comment]
How's this updated version look, @nvkelso?

**Comment by nvkelso** [comment]
LGTM

**Review by nvkelso — APPROVED** [review]

**Pull request merged**